ibm/mas_devops/playbooks/ocp/configure-ocp.yml

```diff
@@ -13,6 +13,14 @@
     artifactory_username: "{{ lookup('env', 'W3_USERNAME') | lower }}"
     artifactory_apikey: "{{ lookup('env', 'ARTIFACTORY_APIKEY') }}"
 
+    # -- Grafana Setup ------------------------------------------------------------------------------------------------
+    grafana_user: "{{ lookup('env', 'GRAFANA_USER') }}"
```

Comment:
Code consistency:

Can we ctrl+h and switch to `grafana_username` / `GRAFANA_USERNAME` 

We already have `bas_grafana_username` / `BAS_GRAFANA_USERNAME` in use in the collection, and it's just asking for trouble not being consistent between the two.